### DISCUSSION TITLE
Recurring issue that may be browser related:

### DISCUSSION THREAD

**Discussion by swamp-cabbage** [vladmandic/sdnext#1398] [General]
I believe that this issue:

[(https://github.com/vladmandic/automatic/issues/340)]
Has affected me again, anyone else ?
I have not changed anything else, my assumption is that Chrome pushed an update which made this happen again.

I turned off image preview and it's fine.

Thanks !

**Comment by ghost** [comment]
I have the same, it was quite bad for a while. img2img especially was bugging for me, had to restart multiple times. Sometimes the gens suddenly all had one third of the it/s in img2img with no controlnet, AND or other slowers activated. After a restart it works again and hasnt come back?

I also get excessively long waiting times at the start or end of an image. not half a second, 10+ seconds. I know this is not of great help to anyone right now but I am too tired to make a proper bug report. Gonna make one later about this, when I can close in on the error a bit

Preview is turned off in my ui completely... but I do have chrome

**Reply by Tillerz** [reply]
Sounds like it runs out of VRAM or something and then moves over to regular memory? Are batches maybe not cleaning up properly?
